invenio_rdm_records/resources/serializers/ui/schema.py

```diff
@@ -143,6 +143,69 @@ class FundingSchema(Schema):
     funder = fields.Nested(FunderL10NItemSchema)
 
 
+def compute_publishing_information(obj, dummyctx):
+    """Computes 'publishing information' string from custom fields."""
+
+    attr = "custom_fields"
+    field = obj.get(attr, {})
+    publication_date = obj.get("metadata", {}).get("publication_date")
+    publisher = obj.get("metadata", {}).get("publisher")
+
+    # Retrieve publishing related custom fields
+    journal = field.get("journal:journal")
+    imprint = field.get("imprint:imprint")
+    thesis = field.get("thesis:university")
+
+    result = {}
+    if journal:
+        title = journal.get("title", "")
+        if title:
+            volume = journal.get("volume", "")
+            issn = journal.get("issn", "")
+            issue = journal.get("issue", "")
+            pages = journal.get("pages", "")
+            journal_string = (
+                "{title}: {volume} ({publication_date}) {issn} {pages} {issue} ".format(
+                    title=title,
+                    volume=volume,
+                    publication_date=publication_date,
+                    issn=f"no. {issn}" if issn else "",
+                    pages=f"pp. {pages}" if pages else "",
+                    issue=f"({issue})" if issue else "",
+                )
+            )
+            result.update({"journal": journal_string})
+
+    if imprint and publisher:
+        place = imprint.get("place", "")
+        isbn = imprint.get("isbn", "")
+        imprint_string = "{publisher}{place} {isbn}".format(
+            publisher=publisher,
+            place=f", {place}",
+            isbn=f"({isbn})"
+        )
+        result.update({"imprint": imprint_string})
+
+    if thesis:
+        result.update({"thesis": thesis})
+
+    return result
+
+
+def compute_conference_information(obj, dummyctx):
```

Comment:
This simply "copies" the conference custom field to another field:

```python
record = {
    'custom_fields': {
        'meeting:meeting': {
            'title': 'xyz'
        }
    },
    'conference': {
        'title': 'xyz'
    }
}
```

and then gets rendered in the UI. 

I didn't create a single string (like in `publishing_information`) because the UI for conference also has other elements (e.g. `url`)